addons/mrp/migrations/10.0.2.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,419 @@
+mrp          / mrp.workorder            / duration_expected (float)     : NEW
+mrp          / mrp.workorder            / final_lot_id (many2one)       : NEW relation: stock.production.lot
+mrp          / mrp.workorder            / message_follower_ids (one2many): NEW relation: mail.followers
+mrp          / mrp.workorder            / message_ids (one2many)        : NEW relation: mail.message
+mrp          / mrp.workorder            / message_last_post (datetime)  : NEW
+mrp          / mrp.workorder            / move_lot_ids (one2many)       : NEW relation: stock.move.lots
+mrp          / mrp.workorder            / move_raw_ids (one2many)       : NEW relation: stock.move
+mrp          / mrp.workorder            / name (char)                   : NEW required: required
+mrp          / mrp.workorder            / next_work_order_id (many2one) : NEW relation: mrp.workorder
+mrp          / mrp.workorder            / operation_id (many2one)       : NEW relation: mrp.routing.workcenter
+mrp          / mrp.workorder            / production_id (many2one)      : NEW relation: mrp.production, required: required
+mrp          / mrp.workorder            / qty_produced (float)          : NEW
+mrp          / mrp.workorder            / qty_producing (float)         : NEW
+mrp          / mrp.workorder            / scrap_ids (one2many)          : NEW relation: stock.scrap
+mrp          / mrp.workorder            / state (selection)             : NEW selection_keys: ['cancel', 'done', 'pending', 'progress', 'ready']
+mrp          / mrp.workorder            / time_ids (one2many)           : NEW relation: mrp.workcenter.productivity
+mrp          / mrp.workorder            / workcenter_id (many2one)      : NEW relation: mrp.workcenter, required: required
+# NOTHING TO DO
+
+mrp          / procurement.order        / property_ids (many2many)      : DEL relation: mrp.property
+# NOTHING TO DO
+
+
+mrp          / stock.move               / consumed_for (many2one)       : DEL relation: stock.move
+# DONE: Pre-migration: Renamed to None and copied this column
+
+mrp          / stock.move               / active_move_lot_ids (one2many): NEW relation: stock.move.lots
+mrp          / stock.move               / bom_line_id (many2one)        : NEW relation: mrp.bom.line
+mrp          / stock.move               / consume_unbuild_id (many2one) : NEW relation: mrp.unbuild
+mrp          / stock.move               / move_lot_ids (one2many)       : NEW relation: stock.move.lots
+mrp          / stock.move               / operation_id (many2one)       : NEW relation: mrp.routing.workcenter
+mrp          / stock.move               / quantity_done_store (float)   : NEW
+mrp          / stock.move               / unbuild_id (many2one)         : NEW relation: mrp.unbuild
+mrp          / stock.move               / unit_factor (float)           : NEW
+mrp          / stock.move               / workorder_id (many2one)       : NEW relation: mrp.workorder
```

Comment:
This one should be set for old workorders not done yet, or better, for all of them for having correctly the history. The same can apply to other fields of this model as well.

Reply:
And through which relation can I fill them?

Reply:
Again I don't know data structure, but I think it should be there in previous version somewhere